mylyn.commons/org.eclipse.mylyn.commons.notifications.tests/META-INF/MANIFEST.MF

```diff
@@ -14,4 +13,5 @@ Require-Bundle: com.sun.xml.bind;bundle-version="2.3.3",
 Export-Package: org.eclipse.mylyn.commons.notifications.tests;x-internal:=true,
  org.eclipse.mylyn.commons.notifications.tests.core;x-internal:=true,
  org.eclipse.mylyn.commons.notifications.tests.feed;x-internal:=true
+Import-Package: com.sun.xml.bind;version="2.3.3"
```

Comment:
I haven’t checked but it would be good it the was not needed at all.

Reply:
I'll have a look how we can eliminate it

Reply:
I'll create another PR to try luck with `com.sun.xml.bind` @merks